aleph/views/serializers.py

```diff
@@ -88,6 +89,13 @@ def jsonify_result(cls, result, extra=None, **kwargs):
         data = result.to_dict(serializer=cls)
         if extra is not None:
             data.update(extra)
+        if data.get("total") > 0 and not data.get("results"):
+            log.exception(f"Expected more results in the response: {data}")
+            data = {
+                "status": "error",
+                "errors": gettext("Failed to load expected results."),
```

Comment:
I was planning to update our error handling in the frontend to always display the error message we got from the API, so in that case it would only display "Failed to load expected results.". Can we update the message to make it clearer that there should be results, but we haven’t been able to load them due to a technical problem?

Reply:
In that case, what do you think about something like: "We found x results, but could not load them due to a technical problem. Please check back later and if the problem persists contact an Aleph administrator"?

Reply:
Yes, totally!